angr/procedures/win32/heap.py

```diff
@@ -13,7 +13,27 @@ def run(self, flOptions, dwInitialSize, dwMaximumSize): #pylint:disable=argument
 
 class HeapAlloc(angr.SimProcedure):
     def run(self, HeapHandle, Flags, Size): #pylint:disable=arguments-differ, unused-argument
-        return self.state.heap._malloc(Size)
+        addr = self.state.heap.malloc(Size)
+
+        # conditionally zero the allocated memory
+        if self.state.solver.solution(Flags & 8, 8):
+            # allocated size might be greater than requested
+            data_size = self.state.solver.eval_one(
+                self.state.heap.chunk_from_mem(addr).get_data_size()
+            )
+            data = self.state.solver.BVV(0, data_size * 8)
+            condition = self.state.solver.If(
+                Flags & 8 == 8,
+                self.state.solver.true,
+                self.state.solver.false
```

Comment:
why did you write it this way? this is equivalent to just `condition=Flags & 8 == 8`

Reply:
Hah, I have no idea how it ended up like that. I'll submit a quick PR.